infra/terraform/modules/tezos_node/user_data.tpl

```diff
@@ -92,8 +92,10 @@ cp /.tezos-${network}/docker-compose.yml .tezos-${network}/docker-compose.yml
 
 echo "> Snapshot rolling-mode node"
 
+current_hash=$(${network}.sh head | tail -n +9 | jq .hash)
+>>> Using the block $${current_hash}
```

Comment:
Why `$$` and not `$` ?

Reply:
It is due to the terraform templatefile. If you don't use the `$$`, it will be interpreted and you will get an empty string in the final version of the file.

Note : it only work for the `${<something>}`. `$(<something)` is not affected